<launch>

    <arg name="path" default="$(find nomask_alarm)/media/"/>

    <node pkg="nomask_alarm" name="nomask_alarm" type="nomask_alarm_node" output="screen" respawn="true">
        <param name="sample_sound1" value="$(arg path)sample1.wav"/>
        <param name="sample_sound2" value="$(arg path)sample2.wav"/>
        <param name="sample_video1" value="$(arg path)sample1.mp4"/>
        <param name="sample_image1" value="$(arg path)sample1.wav"/>
        <param name="sample_image2" value="$(arg path)sample1.wav"/>

        <param name="curse_sound" value="$(arg path)curse.wav"/>
        <param name="curse_sound1" value="$(arg path)curse1.wav"/>
        <param name="curse_sound2" value="$(arg path)curse2.wav"/>
        <param name="curse_video" value="$(arg path)curse.mp4"/>
        <param name="face1" value="$(arg path)face1.png"/>
        <param name="face2" value="$(arg path)face2.png"/>
        <param name="face3" value="$(arg path)face3.png"/>
        <param name="face4" value="$(arg path)face4.png"/>
        <param name="face5" value="$(arg path)face5.png"/>
        <param name="black" value="$(arg path)black.jpg"/>

        <param name="debug" value="false"/>

    </node>

</launch>